ddtrace/contrib/anthropic/patch.py

```diff
@@ -62,7 +64,7 @@ def traced_chat_model_generate(anthropic, pin, func, instance, args, kwargs):
                 if integration.is_pc_sampled_span(span):
                     span.set_tag_str(
                         "anthropic.request.messages.%d.content.0.text" % (message_idx),
-                        integration.trunc(message.get("content", "")),
+                        integration.trunc(message.get("content", "").strip()),
```

Comment:
Why did we add a strip here?

Reply:
A lot of messages come in with weird white spaces and `\n`

Reply:
That's fine, we don't want to modify any existing messages

Reply:
removed all stripping